src/conn/conn_compact.c

```diff
@@ -602,6 +616,17 @@ __background_compact_server(void *arg)
         if (!running)
             continue;
 
+        /*
+         * Throttle background compaction if one of the following conditions is met:
```

Comment:
Change "if one or" to "if any of"

Reply:
Fixed in 42a90cb0823fd76c2ce939d911bd9be8839ae8ed